lib/msal-node/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": "0.0.1",
+  "name": "@azure/msal-node",
+  "author": {
+    "name": "Microsoft",
+    "email": "nugetaad@microsoft.com",
+    "url": "https://www.microsoft.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
+  },
+  "description": "Microsoft Authentication Library for js",
+  "keywords": [
+    "implicit",
+    "js",
+    "AAD",
+    "msal",
+    "oauth"
+  ],
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "tsdx test .*.spec.*",
+    "test:watch": "tsdx test .*.spec.* --watch",
+    "test:coverage": "tsdx test .*.spec.* --coverage",
```

Comment:
I'm going to defer to you on what we use here, but we need to make sure this can be integrated with the coverage of the other libraries. Otherwise we will have to make exceptions for node, which defeats the purpose of this work.

Reply:
Rasied #1372 to track this.